src/i_oplmusic.c

```diff
@@ -1245,24 +1245,11 @@ static void MetaSetTempo(unsigned int tempo)
 static void MetaEvent(opl_track_data_t *track, midi_event_t *event)
 {
     byte *data = event->data.meta.data;
-    unsigned int data_len = event->data.meta.length;
+    uint32_t data_len = event->data.meta.length;
 
     switch (event->data.meta.type)
     {
-        // Things we can just ignore.
-
-        case MIDI_META_SEQUENCE_NUMBER:
-        case MIDI_META_TEXT:
-        case MIDI_META_COPYRIGHT:
-        case MIDI_META_TRACK_NAME:
-        case MIDI_META_INSTR_NAME:
-        case MIDI_META_LYRICS:
-        case MIDI_META_MARKER:
-        case MIDI_META_CUE_POINT:
-        case MIDI_META_SEQUENCER_SPECIFIC:
-            break;
-
-        case MIDI_META_SET_TEMPO:
+        case MIDI_SET_TEMPO:
```

Comment:
I like all the changes in this PR, impressive work! But with so many cosmetic changes, are we giving up sync with Chocolate Doom?

Reply:
That's a good point. I may have gone overboard with the cosmetic changes, honestly. Maybe I should revert some and update Chocolate once things stabilize a bit more (so we don't bug turol too much).

Reply:
Okay, I dropped most of the less important cosmetic changes.

Reply:
Looks good, thank you.